packages/conf-rust-llvm/conf-rust-llvm.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Léo Andrès <contact@ndrs.fr>"
+authors: "Léo Andrès <contact@ndrs.fr>"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+build: [
+  ["which" "rust-lld"] { os-distribution != "homebrew" }
+]
+depends: [
+  "conf-which" {build}
+  "conf-rust"
+  "conf-llvm"
+  "conf-lld"
+]
+depexts: [
+  ["rust-llvm"] { os-family = "alpine" }
+  ["rust-llvm"] { os-family = "arch" }
+  ["rust-llvm"] { os-family = "debian" }
+  ["rust-llvm"] { os-family = "ubuntu" }
+  ["rust-llvm"] { os-family = "opensuse" }
+  ["rust-llvm"] { os-family = "suse" }
+  ["rust-llvm"] { os-distribution = "centos" }
+]
+available: os-distribution != "freebsd" & os-distribution != "fedora"
```

Comment:
```suggestion
```

Unless there are conflicts or severe issues with the possibility of someone hand-crafting an environment where this works.

Reply:
These could be moved to `x-ci-accept-failures` if we want to clean up the CI but not prevent people from satisfying this dependency (and so installing any of its dependencies) in case users manage to build from source or install the system dependency in some other way.

Reply:
Done